patches/chrome-browser-resources-settings-basic_page-basic_page.html.patch

```diff
@@ -21,6 +22,11 @@ index 2e8fb6c622b75e53380e0e21c38e14e116d2c59e..3d0fc996e7e0203a6362b434e96c8139
 +            <settings-default-brave-shields-page  prefs="{{prefs}}"></settings-default-brave-shields-page>
 +          </settings-section>
 +        </template>
++        <!-- brave sync settings -->
```

Comment:
I would prefer not adding comments in the patch, thanks.

Reply:
ok comment removed